packages/material-design-icons/package.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "@automattic/material-design-icons",
+	"version": "1.0.0",
+	"description": "Material icons in SVG format.",
+	"main": "src/index.js",
+	"module": "src/index.js",
+	"sideEffects": false,
+	"keywords": [
+		"wordpress"
+	],
+	"author": "Brad Griffith <brad.griffith@automattic.com>",
+	"contributors": [],
+	"homepage": "https://github.com/Automattic/wp-calypso",
+	"license": "Apache-2.0",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/material-design-icons"
+	},
+	"publishConfig": {
+		"access": "private"
+	},
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"files": [
+		"src"
```

Comment:
should include svg-sprite here

Reply:
Added in ced90eb, which resolves the import alias issue as well.